charts/telepresence/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: telepresence
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 2.2.2
```

Comment:
In my mind, the telepresence binary installing the traffic-agent is still our preferred way of having the traffic-manager installed.  So I'm wondering who this chart is for and what is our responsibility with this chart? 

e.g. Do we have a helm repo we are going to be pushing this chart to? Do we need to bump it whenever we have a new app version or should we just make changes when we change the yaml? Should we be adding linting this to our CI?

Not opposed just want to make sure we have a shared understanding of the purpose of this chart and what our responsibilities are.

Reply:
Discussed with Kenn so mostly have answers about the responsibility (e.g. we should own the chart)

Reply:
This chart is for people who do not wish to/cannot installing software into their cluster that they do not control the YAML for. This can be a common need for users whose environments are locked down by a centralized Ops team and people outside of that Ops team has not access to create cluster-side resources like this. 

This helm chart just creates a way to have an ops team deploy the server-side components using the same mechanisms they do for other things in their environment.